sklearn/preprocessing/_polynomial.py

```diff
@@ -584,6 +584,9 @@ class SplineTransformer(TransformerMixin, BaseEstimator):
     `extrapolation="periodic"`) spline basis functions
     (B-splines) of polynomial order=`degree` for each feature.
 
+    In order to learn more about the SplineTransformer class go to:
+    :ref:`sphx_glr_auto_examples_applications_plot_cyclical_feature_engineering.py`
```

Comment:
This is a good example, and could be mentioned in more places since it's more on the realistic side of things.

I'd suggest having a link in `ColumnTransformer`, `Pipeline`, and `TimeSeriesSplit` as well.

But the description could be more informative. Something like:

Fore a more comprehensive example on a time series data est, see ...

WDYT @glemaitre

Reply:
Definitely for `TimeSeriesSplit`. For `ColumnTransformer` and `Pipeline`, let's do it as well. We have to be careful in the future that for those two, we don't have too many examples.